test/fixtures/query-params.md

```diff
@@ -1,6 +1,6 @@
 # Query params
 
-Link to relative heading ![link](?foo=bar#query-params)
+Link to relative heading [link](?foo=bar#query-params)
```

Comment:
Could this be reverted, unless I’m missing something?

Reply:
Ah, I missed that in your PR body.

> because it is a "Link" not "image", so this looks like a mistake (and prevented tests from working with my code).

It sure looks that way. But, as an image is expected by browsers, they will send something along the lines of an expected `image/png, image/jpg, ...` mimetype, for which the server *could* respond with an image.
I think it’s good to keep it in!

Reply:
Sure, so wouldn't it be better if we create a new case for image and fix the current for heading?
```js
Link to relative heading [link](?foo=bar#query-params) // change to be valid

Link to an ![image](./examples/image.jpg?foo=bar) // current image test without hash

Link to an ![image](./examples/image.jpg?foo=bar#client-params) with hash // new image test with hash
```
to keep test clean and understandable?